src/test/java/io/jenkins/plugins/analysis/warnings/recorder/MiscIssuesRecorderITest.java

```diff
@@ -53,6 +52,14 @@
  */
 @SuppressWarnings({"PMD.ExcessiveImports", "ClassDataAbstractionCoupling"})
 public class MiscIssuesRecorderITest extends IntegrationTestWithJenkinsPerSuite {
+    static final String DETAILS = "Details";
+    static final String FILE = "File";
+    static final String PACKAGE = "Package";
+    static final String CATEGORY = "Category";
+    static final String TYPE = "Type";
+    static final String SEVERITY = "Severity";
+    static final String AGE = "Age";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field AGE has the same name as a method](https://app.codacy.com/app/uhafner/warnings-ng-plugin/pullRequest?prid=4663559)